docs/conf.py

```diff
@@ -1,3 +1,4 @@
+# noqa D103
```

Comment:
I think the ':' is necessary, `# noqa: D103`, but haven't tested.

Reply:
Thanks for pointing this out. I have fixed it everywhere. This noqa stuff is quite new to me.